app/src/test/java/io/intrepid/contest/screens/contestcreation/NewContestPresenterTest.java

```diff
@@ -67,7 +67,7 @@ public void onBackButtonClickedShouldNavigateBackwards() {
     }
 
     @Test
-    public void onBackTwo() {
+    public void onBackTwo(){
```

Comment:
I think in some branch you had removed this test -- it's the same as the previous test